projects/solar/src/fault_handler.c

```diff
@@ -40,7 +40,7 @@ StatusCode fault_handler_raise_fault(EESolarFault fault, uint8_t fault_data) {
     }
   }
 
-  StatusCode can_status = CAN_TRANSMIT_SOLAR_FAULT(fault, fault_data);
+  StatusCode can_status = CAN_TRANSMIT_SOLAR_FAULT_5_MPPTS(fault, fault_data);
```

Comment:
You also need to swap between `SOLAR_FAULT_5_MPPTS` and `SOLAR_FAULT_6_MPPTS` here too. The idea is you should only send `5_MPPT` messages on the 5 MPPT board and `6_MPPT` messages on the 6 MPPT board, otherwise there could be issues.